File: internal/collector/treesitter/testdata/test_kind/java/java-testng/teardown/TeardownTest.java
// SPDX-License-Identifier: Apache-2.0

package fixture;

import org.testng.annotations.AfterClass;
import org.testng.annotations.AfterMethod;

public class TeardownTest {
    @AfterClass
    public void tearDownAll() {}

    @AfterMethod
    public void tearDown() {}
}

File: internal/collector/treesitter/testdata/test_kind/scala/scala-scalatest/teardown/TeardownTest.scala
// SPDX-License-Identifier: Apache-2.0

package fixture

import org.scalatest.funsuite.AnyFunSuite

class TeardownTest extends AnyFunSuite {
  afterAll {
    val _ = 1
  }
  afterEach {
    val _ = 2
  }
}
